File: src/Application Scheduler/ISchedulerForm.cs
using System;
using System.Data;
using System.Windows.Forms;

namespace FSM
{
    public interface ISchedulerForm
    {
        string Name { get; }
        string EngineerID { get; set; }
        bool IsDisposed { get; }
        PictureBox PicPlanner { get; }
        DataTable TimeSlotDt { get; set; }

        string selectedDay();

        IntPtr Handle { get; }
    }
}

File: src/Application Scheduler/ISchedulerForm.vb
Public Interface ISchedulerForm

    ReadOnly Property Name() As String

    Property EngineerID() As String

    ReadOnly Property IsDisposed() As Boolean

    ReadOnly Property PicPlanner() As PictureBox

    Property TimeSlotDt() As DataTable

    Function selectedDay() As String

    ReadOnly Property Handle() As IntPtr

End Interface
